### PR TITLE
Hosted broker logos for Yahoo

### PR DESCRIPTION
**Pull request by jsom**

### CODE REVIEW

File: TradeItIosTicketSDK2/TradeItSelectBrokerViewController.swift [hunk 1]
```diff
@@ -206,7 +206,7 @@ class TradeItSelectBrokerViewController: CloseableViewController, UITableViewDel
             if let broker = broker {
                 let brokerLogoImageView = UIImageView()
                 if TradeItBrokerLogoService.setLogo(forBroker: broker, onImageView: brokerLogoImageView, withSize: .small) {
-                    if let cell = tableView.dequeueReusableCell(withIdentifier: "TRADE_IT_FEATURED_BROKER_CELL_ID") as? TradeItYahooFeaturedBrokerTableViewCell {
+                    if let cell = tableView.dequeueReusableCell(withIdentifier: "TRADE_IT_FEATURED_BROKER_CELL_ID") as? TradeItFeaturedBrokerTableViewCell {
```
Comment (guillaumedebavelaere) [inline TradeItIosTicketSDK2/TradeItSelectBrokerViewController.swift]: Has the storyboard been updated with the custom class TradeItFeaturedBrokerTableViewCell?

File: TradeItIosTicketSDK2/TradeItBrokerLogoService.swift [hunk 2]
```diff
@@ -7,27 +7,18 @@ enum TradeItBrokerLogoSize: String {
 
 class TradeItBrokerLogoService {
     static func setLogo(forBroker broker: TradeItBroker, onImageView imageView: UIImageView, withSize size: TradeItBrokerLogoSize) -> Bool {
-        guard let brokerShortName = broker.brokerShortName else { return false}
-        if let brokerLogoImage = TradeItSDK.brokerLogoService.getLogo(forBroker: brokerShortName) {
```
Comment (guillaumedebavelaere) [inline TradeItIosTicketSDK2/TradeItBrokerLogoService.swift]: So we are not using anymore the "local" logos ?

Comment (jsom) [inline TradeItIosTicketSDK2/TradeItBrokerLogoService.swift]: Yeah - it'll all be managed by us now.

File: TradeItIosTicketSDK2/TradeItSelectBrokerViewController.swift (hunk 1 above)
Comment (jsom) [inline TradeItIosTicketSDK2/TradeItSelectBrokerViewController.swift]: Done

**Pull request merged**